tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/TeamMailboxMemberGetMethodContract.scala

```diff
@@ -0,0 +1,310 @@
+package com.linagora.tmail.james.common
+
+import com.google.common.collect.ImmutableList
+import com.linagora.tmail.team.{TeamMailbox, TeamMailboxName, TeamMailboxProbe}
+import eu.timepit.refined.auto._
+import io.netty.handler.codec.http.HttpHeaderNames.ACCEPT
+import io.restassured.RestAssured.{`given`, requestSpecification}
+import io.restassured.http.ContentType.JSON
+import net.javacrumbs.jsonunit.JsonMatchers.jsonEquals
+import net.javacrumbs.jsonunit.core.Option.IGNORING_ARRAY_ORDER
+import org.apache.http.HttpStatus.SC_OK
+import org.apache.james.GuiceJamesServer
+import org.apache.james.jmap.core.ResponseObject.SESSION_STATE
+import org.apache.james.jmap.http.UserCredential
+import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, ANDRE, ANDRE_PASSWORD, BOB, BOB_PASSWORD, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.utils.DataProbeImpl
+import org.junit.jupiter.api.{BeforeEach, Test}
+
+trait TeamMailboxMemberGetMethodContract {
```

Comment:
Please add a test: bob tries to list members of a team mailbox he do not have access to.